<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KaitaiStructForm</class>
 <widget class="QWidget" name="KaitaiStructForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_ksy">
      <attribute name="title">
       <string>KSY</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabExecOptions">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_precompiled">
          <attribute name="title">
           <string>Precompiled</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="pb_selectPrecompiled">
             <property name="text">
              <string>Select structure...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>571</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_custom">
          <attribute name="title">
           <string>Custom</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <property name="labelAlignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>ksc:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>le_ksc</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLineEdit" name="le_ksc">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="tb_chooseKsc">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pb_loadKsy">
               <property name="text">
                <string>Load existing ksy...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Enter Kaitai Struct Definition (ksy):</string>
               </property>
               <property name="buddy">
                <cstring>te_ksy</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="te_ksy">
               <property name="documentTitle">
                <string/>
               </property>
               <property name="lineWrapMode">
                <enum>QPlainTextEdit::NoWrap</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_output">
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Program Output:</string>
           </property>
           <property name="buddy">
            <cstring>te_output</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="te_output">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="undoRedoEnabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Parsed</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="layout_nav"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_ksc</tabstop>
  <tabstop>tb_chooseKsc</tabstop>
  <tabstop>te_ksy</tabstop>
  <tabstop>te_output</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
